charts/redpanda/templates/secrets.yaml

```
@@ -91,3 +92,21 @@ stringData:
         sleep 0.5
     done
 {{- end }}
+{{- if and .Values.auth.sasl.enabled ( gt ( len .Values.auth.sasl.users ) 0 ) }}
+---
+apiVersion: v1
```

If the user is going to manage this secret manually, they need a way to prevent the helm chart from failing when it tries to create the same secret.
```suggestion
{{- if .Values.auth.sasl.createSecret }}
apiVersion: v1
```